.claude/commands/agents/gemini/gemini-agent.md

```
@@ -0,0 +1,367 @@
+<bash_tool_params>
+command: "gemini -p {task_prompt} -m {model_name} > {task_id_output_path}"
+description: "Builds the project"
+timeout: 600000  # 5 minutes
+</bash_tool_params>
+
+<context_passing_options>
+When executing Gemini commands, you can provide file context in two ways:
+
+1. Use the -a flag to pass the context of ALL files in the current directory:
+   `gemini -a -p "{task_prompt}" -m {model_name}`
+   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)